### ISSUE TITLE
Inference on arbitrary images

### ISSUE THREAD

**Issue by ogencoglu** [weiliansong/blueprint-vectorizer#1]
Thanks for the interesting work.

Would it be possible to run the model inference without having to have ground truth labels. After all, the data is not shared and all we can do is to run on our images.

**Comment by weiliansong** [comment]
Yes that's a fair request. Will try to remove the label dependency either this or next week, currently a bit tied up now

**Comment by ogencoglu** [comment]
Thanks a lot @weiliansong !

**Comment by weiliansong** [comment]
Still in the process of removing GT label dependencies, but the first two stages are done. See 07ebef320cb508aa1413d1dcf182fc35d7a98ae6.

**Comment by weiliansong** [comment]
I also did run the pipeline with some blueprint-like images on the internet. Unfortunately the performance from the instance segmentation stage is not good, as our dataset is quite specialized. So unless your data looks very similar to ours, you will most likely have to retrain.
